### PR TITLE
[SAM-866] Update Contributing.md file

### PR DESCRIPTION
**Pull request by Carryace**
## Description
Update the steps in CONTRIBUTING.md file to set up environment locally. 

##JIRA Ticket
http://gsaiae.jira.micropaas.io:8080/browse/SAM-866

### CODE REVIEW

**Comment by Carryace** [comment]
@joshbruce - The steps to solve the problem works for mac, Justin and I have verified it.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -140,6 +140,26 @@ Note: For a list of available gulp commands use:
 
 `$ gulp`
 
+#### Trouble shooting(For Mac)
+
+If you have problem serve the site locally after all these steps, please try the following steps:
```
Comment (joshbruce) [inline CONTRIBUTING.md]: Change to:

If you have a problem serving the site locally, please try the following:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -140,6 +140,26 @@ Note: For a list of available gulp commands use:
...
+
+Launch Terminal (OS X) and [install RVM](https://rvm.io/rvm/install)
```
Comment (joshbruce) [inline CONTRIBUTING.md]: Change "OS X" to "MacOS" - Apple recently rebranded the operating system.

These instructions do not appear to be limited to MacOS. Therefore, maybe we only need the link to the installation instructions...??

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -140,6 +140,26 @@ Note: For a list of available gulp commands use:
...
+
+Find a stable version of ruby and install it:
+
+`$ rvm install ruby-VersionNumber`
```
Comment (joshbruce) [inline CONTRIBUTING.md]: Replace `ruby-VersionNumber` with `[version number]`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -140,6 +140,26 @@ Note: For a list of available gulp commands use:
...
+
+Use a stable version of ruby for rvm:
+
+`$ rvm use VersionNumber`
```
Comment (joshbruce) [inline CONTRIBUTING.md]: Replace `ruby-VersionNumber` with `[version number]`

File: CONTRIBUTING.md (hunk 3 above)
Comment (Carryace) [inline CONTRIBUTING.md]: Here is an example of this command: rvm install ruby-2.3.1. So, I think we should change to $rvm install ruby-[version number]

File: CONTRIBUTING.md (hunk 4 above)
Comment (Carryace) [inline CONTRIBUTING.md]: An example for this: rvm use 2.3.1. I will change this command to: "$rvm use [version-number]"

**Comment by Carryace** [comment]
@joshbruce - It should be a small part of the third section, so we don't need to add the section to TOC menu.

**Pull request merged**